README.md

```diff
@@ -1,12 +1,25 @@
 # intershop.github.io
 
-## Intershop XSD  ##
+Welcome to **intershop.github.io**! 
 
-New Intershop XSD provide semantic versioning. The XSD containing the major version in their namespace.
-Additionally former XSD are provided to identify differences between versions.
+This repository contains XSD (XML Schema Definition) files that facilitate the creation of XML files for defining process chains. Process chains are a mechanism for automating the sequential execution of interdependent scheduled jobs or pipelines in Intershop Commerce Management.
 
-| file location       | target namespace                             |
-|---------------------|----------------------------------------------|
-| xml/ns/semantic     | https://www.intershop.com/xml/ns/semantic    |
-| xml/ns/enfinity/6.4 | http://www.intershop.com/xml/ns/enfinity/6.4 |
-| xml/ns/enfinity/6.5 | http://www.intershop.com/xml/ns/enfinity/6.5 |
+For more information on process chains and their usage, refer to the following document in the Intershop Knowledge Base: 
+- [Overview - Process Chains](https://support.intershop.com/kb/index.php/Display/2784F3)
+
+## Semantic Versioning
+
+Newer versions of the XSD files follow semantic versioning principles. Each XSD file's namespace specifies its version.
```

Comment:
May be reworded to "Newer versions of the XSD files including the `semantic` keyword"